ng2/5-piratenameservice/lib/src/name_service.dart

```diff
@@ -12,16 +12,31 @@ class NameService {
   static final Random _indexGen = new Random();
 
   final _names = <String>[
-    'Anne', 'Mary', 'Jack', 'Morgan', 'Roger',
-    'Bill', 'Ragnar', 'Ed', 'John', 'Jane'
+    'Anne',
```

Comment:
This is the one place where I really don't like the dartfmt output. It really bloats this code.